lib/services/discourse-types.d.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    ServiceEmitContext,
+} from './service-types';
+
+export interface DiscourseMessageEmitContext extends ServiceEmitContext {
+    raw: string,
```

Comment:
Sounds possibly. If this is the plugin extension (https://marketplace.visualstudio.com/items?itemName=eg2.tslint) then maybe the `ignoreDefinitionFiles` boolean is set to true by default (handily, it doesn't say on that page!)?